main/MiNiFiMain.cpp

```diff
@@ -142,94 +159,39 @@ int main(int argc, char **argv) {
 #ifdef WIN32
 	if (!SetConsoleCtrlHandler(consoleSignalHandler, TRUE)) {
 		logger->log_error("Cannot install signal handler");
-		std::cerr << "Cannot install signal handler" << std::endl;
-		return 1;
+		return -1;
 	}
 
 	if (signal(SIGINT, sigHandler) == SIG_ERR || signal(SIGTERM, sigHandler) == SIG_ERR ) {
-		std::cerr << "Cannot install signal handler" << std::endl;
+		logger->log_error("Cannot install signal handler");
 		return -1;
 	}
 #ifdef SIGBREAK
 	if (signal(SIGBREAK, sigHandler) == SIG_ERR) {
-		std::cerr << "Cannot install signal handler" << std::endl;
+		logger->log_error("Cannot install signal handler");
 		return -1;
 	}
 #endif
 #else
 	if (signal(SIGINT, sigHandler) == SIG_ERR || signal(SIGTERM, sigHandler) == SIG_ERR || signal(SIGPIPE, SIG_IGN) == SIG_ERR) {
-		std::cerr << "Cannot install signal handler" << std::endl;
+    logger->log_error("Cannot install signal handler");
 		return -1;
 	}
 #endif
-	// assumes POSIX compliant environment
```

Comment:
The original order of sources for MINIFI_HOME here was
for *nix: MINIFI_HOME env -> realpath(argv[0]) (path of the current executable) -> cwd
for Windows: MINIFI_HOME env -> GetModuleHandle (path of the current executable) -> cwd
This has been cleaned up and unified in `determineMinifiHome` as:
 MINIFI_HOME env -> path of the current executable -> cwd
The method used for determining the path of the current executable used now is superior to `realpath(argv[0])`